templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -1458,13 +1461,7 @@ Resources:
             EOF
             popd
             pushd "$tap_dir/src/resources"
-            aws s3 cp --no-progress "${QSS3BucketPath}/src/resources/developer-namespace.yaml" ./developer-namespace.yaml
```

Comment:
all these s3 cp are required. Is this an oversight?

Reply:
The individual copies have been replaced with a "wildcard copy", see: https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/pull/11/files#diff-0d891330f3a704011d4ebc74cd8faaf8a91d3431e2a5fdcfbcf9a8be544b8515R1464

Reply:
I get it.  This is fine for now.
FYI, we have some files like git-ssh-basic-auth.yaml & workload-aws.example.yaml not used in the code.